src/browser/provider/built-in/dedicated/chrome/cdp-client/emitters/client-request.ts

```diff
@@ -0,0 +1,61 @@
+import EventEmitter from '../../../../../../../shared/utils/event-emitter';
+import { ClientReqEmitter } from '../../../../../../../shared/types';
+import ProtocolProxyApi from 'devtools-protocol/types/protocol-proxy-api';
+import NetworkApi = ProtocolProxyApi.NetworkApi;
+import Protocol from 'devtools-protocol';
+import RequestWillBeSentEvent = Protocol.Network.RequestWillBeSentEvent;
+import LoadingFinishedEvent = Protocol.Network.LoadingFinishedEvent;
+import LoadingFailedEvent = Protocol.Network.LoadingFailedEvent;
+
+
+type ClientReqType = string;
```

Comment:
The such reductions (`ClientReqType`, `ClientReqEvtListener`) make the code readability worse. 
Please, use the whole words in naming in all places (filenames, variables, types, etc.)